src/model_clay.py

```diff
@@ -773,15 +772,41 @@ def clay_large(**kwargs):
         "decoder_mlp_ratio": 4,
         "decoder_dropout": 0.0,
     }
-    model = CLAY(**args.update(kwargs))
+    args.update(kwargs)
+    model = CLAY(**args)
     return model
 
 
 class CLAYModule(L.LightningModule):
-    def __init__(self, lr=1e-4, wd=0.05, b1=0.9, b2=0.95):
+    def __init__(
```

Comment:
Add this to silence the [PLR0913 Too many arguments](https://docs.astral.sh/ruff/rules/too-many-arguments) in function definition (9 > 6) warning
```suggestion
    def __init__(  # noqa: PLR0913
```

Reply:
Thanks!